specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -1,3215 +1,3193 @@
-					},
-					{
-						"$ref": "#/parameters/externalSubscriptionNameParameter"
-					}
-				],
-				"responses": {
-					"200": {
-						"description": "OK. The request has succeeded.",
-						"schema": {
-							"$ref": "#/definitions/ExternalSubscriptionDefinition"
-						}
-					},
-					"default": {
-						"description": "Error response describing why the operation failed.",
-						"schema": {
-							"$ref": "#/definitions/ErrorResponse"
-						}
-					}
-				}
-			}
-		},
-		"/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/Microsoft.CostManagement/externalSubscriptions": {
-			"get": {
-				"tags": [
-					"ExternalSubscriptions"
-				],
-				"operationId": "ExternalSubscription_ListByManagementGroup",
-				"description": "List all ExternalSubscription definitions for Management Group",
-				"externalDocs": {
-					"url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
-				},
-				"x-ms-examples": {
-					"ExternalSubscription_List": {
-						"$ref": "./examples/ManagementGroupExternalSubscriptionList.json"
-					}
-				},
-				"parameters": [{
-						"$ref": "#/parameters/apiVersionParameter"
-					},
-					{
-						"$ref": "#/parameters/managementGroupIdParameter"
-					}
-				],
-				"responses": {
-					"200": {
-						"description": "OK. The request has succeeded.",
-						"schema": {
-							"$ref": "#/definitions/ExternalSubscriptionDefinitionListResult"
-						}
-					},
-					"default": {
-						"description": "Error response describing why the operation failed.",
-						"schema": {
-							"$ref": "#/definitions/ErrorResponse"
-						}
-					}
-				},
-				"x-ms-pageable": {
-					"nextLinkName": null
-				}
-			}
-		},
-		"/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/Microsoft.CostManagement/externalSubscriptions/{externalSubscriptionName}": {
-			"put": {
-				"tags": [
-					"ExternalSubscriptions"
-				],
-				"operationId": "ExternalSubscription_UpdateByManagementGroup",
-				"description": "Associates ExternalSubscription with the management group",
-				"externalDocs": {
-					"url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
-				},
-				"x-ms-examples": {
-					"ManagementGroupExternalSubscription_Put": {
-						"$ref": "./examples/ManagementGroupExternalSubscriptionPut.json"
-					}
-				},
-				"parameters": [{
-						"$ref": "#/parameters/apiVersionParameter"
-					},
-					{
-						"$ref": "#/parameters/managementGroupIdParameter"
-					},
-					{
-						"$ref": "#/parameters/externalSubscriptionNameParameter"
-					}
-				],
-				"responses": {
-					"204": {
-						"description": "NoContent"
-					},
-					"default": {
-						"description": "Error response describing why the operation failed.",
-						"schema": {
-							"$ref": "#/definitions/ErrorResponse"
-						}
-					}
-				}
-			}
-		},
-		"/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/Microsoft.CostManagement/getEntities": {
```

Comment:
Removing this API from an existing API version is considered a breaking change. Have you verified that this URL is not handling any traffic? Have client SDKs been generated for this API version?

Reply:
This URL was never available in production. No client SDKs were generated with this API.